examples/read_bids_datasets.py

```diff
@@ -0,0 +1,93 @@
+"""
+=========================================
+02. Read and load datasets in BIDS format
```

Comment:
isn't there already an example which is 02.?

Reply:
@adam2392 do ping us when you address the reviews because we don't know if you push the commits

Reply:
@jasmainak should I renumber all the examples then manually? If so, that seems a bit unwieldy. Is there a better way to do it? Is there a specific numbering to all of them to use?

Reply:
I don't think so. For now, it's not a problem since we only have a few examples :)